guides/source/rails_application_templates.md

```diff
@@ -32,16 +32,16 @@ $ rails app:template LOCATION=http://example.com/template.rb
 Template API
 ------------
 
-The Rails templates API is easy to understand. Here's an example of a typical Rails template:
+Here's an example of a Rails template:
 
 ```ruby
 # template.rb
 generate(:scaffold, "person name:string")
-route "root to: 'people#index'"
+route "root 'people#index'"
+rails_command("db:create")
```

Comment:
The database needs to be created before running  `db:migrate`.  The example seems more complete to me with this command added.